src/System.Security.Cryptography.Encoding/tests/TestHelpers.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System
+{
+    // @todo: https://github.com/dotnet/corefx/issues/26894 - these emulate MemoryExtension apis that we removed. Clean up the callsites and remove this class.
```

Comment:
I'd prefer it if there wasn't the need for these TestHelpers; but if you really have to do this in stages, can you create a tracking bug for the removal of these so it doesn't get lost before we ship?

Reply:
I'm using the issue itself as the tracking issue. It's just part of the other tailing work like removing the members from the CoreCLR implementation. It'll just no longer be a "3/7 or bust" issue since the apis are removed from the reference assemblies.